### ISSUE TITLE
Odoo14 account_reconciliation_widget is showing blank  cell_label state.st_line.payment_ref for me.

### ISSUE THREAD

**Issue by whotopia** [OCA/account-reconcile#535]
I have a bank statement with many lines to reconcile.  The statement has text about the transaction in the payment_ref field "Label".    However, once the statement lines are posted, and I try to reconcile them within the  account_reconciliation_widget there is no text in the grey field of the bank statement.  However, right before and after, the date and amount are correct.

## Module

account_reconciliation_widget  

## Describe the bug
This behaviour is consistent for me whenever I tried to use the reconciliation widget.  Given that no one else here seems to have mentioned this perhaps it could be due to some other consideration?  (access rights? language setting?  anything else?)

## To Reproduce

Odoo 14  CE

Steps to reproduce the behaviour:
1.  Install the addon 
2. Create a bank statement with text in the Label (payment_ref) field along with anything else needed in order for the line to be valid.   By the way, in my case, the Reference and Notes fields are typically blank
3. Go to the reconciliation widget after posting the statement lines.   You will see that there is no preview of the text from the payment_ref field.  

![Screenshot 2023-03-09 at 14 18 12](https://user-images.githubusercontent.com/9039050/224035007-ad37e874-8aa3-4882-b501-7a0b4206b005.png)

**Expected behaviour**
The field in grey should contain whatever the text is from the bank statement line payment_ref.   Instead it is blank.

**Additional context**

Running Odoo 14 CE with this addon.   Using Python 3.8.13 

I am also using multiple currencies and also the accounting addons from CybroSys.  In this particular case the account in question is for my company native currency .

**Comment by pedrobaeza** [comment]
Is this reproducible in runboat?

**Comment by whotopia** [comment]
> Is this reproducible in runboat?

I tried on RunBoat with https://runboat.odoo-community.org/webui/builds.html?repo=oca/account-reconcile&target_branch=14.0

Created a similar scenario minus multi-currency and main currency being non-USD.     It works fine.

So I guess some other addon is messing with things?

**Comment by pedrobaeza** [comment]
Yes, I'm afraid so, or maybe outdated code.

Closing this, but comment if you are able to reproduce it.

**Comment by whotopia** [comment]
Thanks for quick reply @pedrobaeza .   I dug around a bit more and I have concluded that there's indeed a negative interaction between the account_reconciliation_widget and the CybrOdoo/CybroAddons/base_accounting_kit.   When these two addons (most recent versions (Odoo V14) from Github + OCA/OCB) are co-installed, the problem is readily reproducible.   The CybroSys addon seems to break the account_reconciliation_widget.    I have alerted CybroSys via https://github.com/CybroOdoo/CybroAddons/issues/245
